src/main/pg/motor.h

```diff
@@ -23,30 +23,45 @@
 #include "pg/pg.h"
 
 #include "drivers/io.h"
-#include "drivers/dshot_bitbang.h"
 
+#if !defined(BRUSHED_MOTORS_PWM_RATE)
+#define BRUSHED_MOTORS_PWM_RATE 16000
```

Comment:
THis is used only once, in pg/motor.c. But maybe keep it here for consistency with BRUSHLESS_MOTORS_PWM_RATE